.gitchangelog.rc

```diff
@@ -83,15 +83,14 @@ ignore_regexps = [
 ##
 section_regexps = [
     ('New', [
-        r'^[nN]ew\s*:\s*((dev|use?r|pkg|test|doc)\s*:\s*)?([^\n]*)$',
-     ]),
-    ('Changes', [
-        r'^[cC]hg\s*:\s*((dev|use?r|pkg|test|doc)\s*:\s*)?([^\n]*)$',
+        r'^([nN]ew|[fF]eat|[aA]dd)\s*(:|)\s*([^\n]*)$',
```

Comment:
👀  I'll follow this format in my following PRs.

Reply:
If you follow semantic pull request, `feat` will be here 🆗